internal/dev_server/dev_server.go

```diff
@@ -57,6 +57,7 @@ func (c LDClient) RunServer(ctx context.Context, accessToken, baseURI, devStream
 	handler = handlers.CombinedLoggingHandler(os.Stdout, handler)
 	handler = handlers.RecoveryHandler(handlers.PrintRecoveryStack(true))(handler)
 	fmt.Println("Server running on 0.0.0.0:8765")
+	fmt.Println("Access the UI for toggling overrides at http://localhost:8765/ui or by running `ldcli dev-server ui`")
```

Comment:
```suggestion
	fmt.Println("✨✨✨ Access the UI for toggling overrides at http://localhost:8765/ui or by running `ldcli dev-server ui` ✨✨✨")
```

Reply:
@mike-zorn oh I've tried to get emoji in the CLI before trust me - but apparently [there's a risk](https://darrenburns.net/posts/emoji-in-the-terminal/) that some terminals may not support it 😭

Reply:
😭😭😭😭😭😭😭😭😭😭😭😭😭😭😭😭😭😭😭😭😭😭😭😭😭😭😭😭😭😭😭😭😭😭😭😭😭😭😭😭😭😭😭😭😭😭😭😭😭😭😭😭😭😭😭😭😭😭😭😭😭😭😭😭😭😭😭😭😭😭😭😭😭😭😭😭😭😭😭😭😭😭😭😭😭😭😭😭